prov/efa/src/rxr/rxr_ep.c

```diff
@@ -998,10 +998,11 @@ static int rxr_ep_ctrl(struct fid *fid, int command, void *arg)
 		 * shared memory region.
 		 */
 		if (ep->use_shm) {
-			ret = rxr_raw_addr_to_smr_name(ep->core_addr, shm_ep_name);
+			ret = rxr_raw_addr_to_smr_name(ep->core_addr, &shm_ep_name);
```

Comment:
what's the issue to use single pointer here?

Reply:
when using single pointer, the caller need to prepare memory, but the caller does not know how much memory to prepare.

Reply:
I see, thanks.